essm/variables/units.py

```diff
@@ -89,10 +89,19 @@ def derive_unit(expr, name=None):
     """Derive SI-unit from an expression, omitting scale factors."""
     from essm.variables import Variable
     from essm.variables.utils import extract_variables
+    from sympy.physics.units import Dimension
     from sympy.physics.units.dimensions import dimsys_SI
+
+    SI_DIMENSIONS = {str(Quantity.get_dimensional_expr(d)):
```

Comment:
do you need to redefine it (see line 38)?

Reply:
Good point, thanks! I removed it.